Rollbar-Notifications-Rule/inputs/inputs_1_create.json

```diff
@@ -0,0 +1,18 @@
+{
+  "ProjectAccessToken": "1df75c0dc67845a780ceda737ae91bc4",
```

Comment:
Could this be parameterized to make it easier to know what needs modified?

Reply:
@joshuadeanhall Done